history-network.md

```diff
@@ -76,25 +76,58 @@ serialized-content-key = serialize(content-key)
 ```
 selector     = 0x00
 content-key  = Container(chain-id: uint16, block-hash: Bytes32)
-content       = rlp(header)
+content      = rlp(header)
 ```
 
 #### Block Body
 
 ```
-selector     = 0x01
-content-key  = Container(chain-id: uint16, block-hash: Bytes32)
-content      = rlp([transaction_list, uncle_list])
+selector                = 0x01
+content-key             = Container(chain-id: uint16, block-hash: Bytes32)
+content                 = Container(all-transactions, serialized-uncles)
+all-transactions        = SSZList(serialized-transaction, max-length=2**14)
+serialized-transaction  = SSZList(encoded-transaction: Byte, max-length=2**24)
+encoded-transaction     =
```

Comment:
I think the way this is written down now, with the usaged of both encoded and serialized wording, it is perhaps a bit difficult to follow? I don't have an better suggestion ready though.

Reply:
Yeah, I'm not really happy with those names either. Do you think `ssz-transaction` and `encoded-transaction` are any clearer?